source/moto_payments/index.html.md.erb

```diff
@@ -16,43 +16,66 @@ This means you can take payments:
 
 These payments are also known as Mail Order / Telephone Order (MOTO) payments.
 
-You cannot take MOTO payments if your payment service provider (PSP) is SmartPay.
+You cannot take MOTO payments:
 
-You also cannot take MOTO payments through a [payment link](https://www.payments.service.gov.uk/govuk-payment-pages/) - [contact us](/support_contact_and_more_information/#contact-us) if you'd like us to develop this feature.
+- if your payment service provider (PSP) is Smartpay
+- through a [payment link](https://www.payments.service.gov.uk/govuk-payment-pages/)
 
-## If your PSP is Worldpay or ePDQ
+Our product roadmap is based on feedback. [Contact us](/support_contact_and_more_information/#contact-us) if you'd like us to investigate taking MOTO payments through payment links.
```

Comment:
'Our product roadmap shows what features we're delivering or exploring, and it's based on feedback from services on their needs.' 

Might be the case that not everyone understands what a roadmap is or how feedback is used (particularly in local gov).

Reply:
Could delete the first sentence, and make it just a call to action.